hue/test_hue.py

```diff
@@ -60,6 +60,8 @@ def test_hue(self, configuration, machine_suffixes):
         'STANDARD',
     )
     def test_hue_job(self, configuration):
+        if self.getImageVersion() == pkg_resources.parse_version("2.2"):
```

Comment:
Can you make this change future-proof? Check >=2.2 instead of 2.2.

Reply:
done